src/sections/pools/pool/incentives/row/MultiplePoolIncentivesRow/MultiplePoolIncentivesRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { Text } from "components/Typography/Text/Text"
+import { Separator } from "components/Separator/Separator"
+import FarmAssetIcon from "./FarmAssetIcon"
+import { AprFarm, getMinAndMaxAPR } from "utils/apr"
+import { useTranslation } from "react-i18next"
+
+type Props = {
+  farms: AprFarm[]
+}
+
+const MultiplePoolIncentivesRow = ({ farms }: Props) => {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <div sx={{ flex: "row", justify: "space-between" }}>
+        <div sx={{ flex: "row" }}>
+          {farms.map((farm, index) => (
+            <FarmAssetIcon
+              key={String(farm.assetId)}
```

Comment:
jen at drzime stejnou notaci

```suggestion
              key={farm.assetId.toString()}
```